tests/regression/test_tzkt_no_content_request.py

```diff
@@ -10,10 +10,39 @@ def test_request_no_content_response():
 
     # The baker address exists **only** on the mainnet
     baker_address = "tz1NortRftucvAkD1J58L32EhSVrQEWJCEnB"
-    base_url = "https://api.carthage.tzkt.io/v1/"
+    base_url = "https://api.delphinet.tzkt.io/v1/"
```

Comment:
shouldn't it be something like "https://api.edo2net.tzkt.io/v1/" ? Bad to have tests that are depending on moving targets. I am not sure how long those addresses for old networks will be maintained?

Reply:
The test tests if the baker which only exists on the mainnet exists on delphinet which he doesn't thus throwing the 204 response. https://api.edo2net.tzkt.io/v1/ would give an OK (200) response.

Reply:
@jdsika has a point. Delphinet will close after some time. Can we use a dummy address for this purpose?

Reply:
done